src/components/app/DefinitionPane/DefinitionPane.mobile.tsx

```diff
@@ -45,7 +45,7 @@ const DefinitionPane: React.SFC<DefinitionPane.Props> = (props) => {
     <StyledDefinitionPane>
       <Upper
         badges={''}
-        author={Definition.get('username')}
+        author={Definition.get('$user')}
```

Comment:
name of property to be `User`, this is in order to let know it is wrapped object